client/src/partials/templates/bhLatestInvoice.tmpl.html

```diff
@@ -14,8 +14,11 @@
     </span>
   </div>
   <div class="panel-body">
+    <div ng-if="$ctrl.loading" class="text-center alert alert-info fade in">
```

Comment:
I wouldn't use an alert here.  An `alert` should be very specific to get a user's attention, but this is just trying to mask a panel while it loads.

Check out the styles for [grid-msg-overlay](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/css/ui-grid.css#L20).  I think it does a good job of hiding the underlying contents without grabbing the user's attention.